homeassistant/components/inels/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "inels",
+  "name": "iNELS",
+  "codeowners": ["@Nejezchleb", "@jhoralek", "@zed4805"],
+  "config_flow": true,
+  "dependencies": ["mqtt"],
+  "documentation": "https://github.com/zed4805/inels-hacs-new",
+  "iot_class": "local_push",
+  "issue_tracker": "https://github.com/zed4805/inels-hacs-new/issues",
```

Comment:
I belive issue_tracker and documenation are not keys that should be passed. Those should only be passed in HACS integrations

Reply:
This is actually wrong, there is a use for both of these keys:

https://developers.home-assistant.io/docs/creating_integration_manifest/#documentation
https://developers.home-assistant.io/docs/creating_integration_manifest/#issue-tracker

Reply:
Keep reading that documentation

If this integration is being submitted for inclusion in Home Assistant, it should be https://www.home-assistant.io/integrations/<domain>

If this integration is being submitted for inclusion in Home Assistant, it should be omitted. For built-in integrations, Home Assistant will automatically generate the correct link.

They shouldn't be linking to hacs

Reply:
Thank you, I've changed it